### ISSUE TITLE
Show Data Download

### ISSUE THREAD

**Issue by RussellAult** [matt-dale/designdbonline#47]
This is probably a gargantuan task, but it would be great if there was some way to download a whole show as a flat file, especially since the project is still in beta (create once, backup twice...).

Thanks!

**Comment by matt-dale** [comment]
So you want under the Equipment Lists tab, just download the every report there with one button?  That shouldn't be hard at all

**Comment by RussellAult** [comment]
Not quite; what I was hoping for is a way to download the raw data of a show in such a way that it could then be uploaded into some other instance of the software. That way I could take the show I just built online, back it up in a ready-to-be-edited-again format (just in case your hosting company does something stupid), and possibly even migrate it to, say, a self-hosted offline installation for spaces with no Internet access. If all I have are the reports, I would have to rebuild the show cable by cable before I could make any edits.

Does that make more sense?

**Comment by RussellAult** [comment]
I've added this as a suggestion on the new project as well.
